tsdat/qc/checkers.py

```diff
@@ -55,7 +55,7 @@ def run(self, dataset: xr.Dataset, variable_name: str) -> NDArray[np.bool_]:
 class CheckMonotonic(QualityChecker):
```

Comment:
Did you check that these changes work on a real datafile? I don't trust the short tests enough...

Reply:
I updated the tests for `CheckMonotonic` using a 1D variable after noticing that the parts I updated here were not covered by the old tests. I'll test it with a 2D variable also to be safe.

Reply:
@jmcvey3 I added another few tests for `CheckMonotonic` and fixed its behavior for 2D variables